app/javascript/app/components/button/button-component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import styles from './button-styles.scss';
+
+const Button = (props) => {
```

Comment:
I like the idea of having a button and pass as children the icon or the text!
👏